lib/surveyor/models/question_methods.rb

```diff
@@ -13,12 +13,20 @@ module QuestionMethods
 
       included do
         # Associations
-        belongs_to :survey_section, required: false
-        belongs_to :question_group, dependent: :destroy, required: false
-        has_many :answers, -> { order('display_order, id ASC') }, dependent: :destroy, autosave: true # it might not always have answers
+        belongs_to :survey_section, optional: true
+        belongs_to :question_group, dependent: :destroy, optional: true
+        has_many :answers,
+          -> { order('display_order, id ASC') },
+          dependent: :destroy,
+          autosave: true # it might not always have answers
+
         has_one :dependency, dependent: :destroy
-        belongs_to :correct_answer, class_name: 'Answer', dependent: :destroy, required: false
-        attr_accessible *PermittedParams.new.question_attributes if defined? ActiveModel::MassAssignmentSecurity
+        belongs_to :correct_answer, class_name: 'Answer', dependent: :destroy, optional: true
```

Comment:
BTW by moving these to `optional` I believe you're removing support for 4.2 as I don't believe this keyword exists there.

Reply:
I guess I can cut that off too then, I don't honestly see much reason to care about unsupported rails anyway

Reply:
I agree -- just making the note.